test/integration/targets/win_psmodule/defaults/main.yml

```diff
@@ -1,7 +1,27 @@
 ---
-
 powershell_module: powershell-yaml
 wrong_module: powershell_yaml
 allow_clobber_module: PowerShellCookbook
+fake_repo_name: FakeRepo
 custom_repo_path: C:\_repo
 custom_repo_name: PSRegisterRepo
+
+powershell_module_required_version: 0.3.4
+powershell_module_maximum_version_request: 0.2.2
+powershell_module_maximum_version_result: 0.2
+powershell_module_ultra_high_version: 15.0.0
+powershell_module_ultra_low_version: 0.0.1
+
+skip_publisher_check_module: Pester
+
+powershell_module_2: Pester
+powershell_module_2_minimum_version_request: 3.1.2
+powershell_module_2_maximum_version_request: 3.3.0
+powershell_module_2_minimum_maximum_version_result: 3.2.0
+
+allow_prerelease_module: Pester
+allow_prerelease_version: 4.4.0-beta2
+
+myget_powershell_module: xActiveDirectory
+myget_repository_name: My Get
+myget_repository_url: https://www.myget.org/F/powershellgetdemo/api/v2
```

Comment:
This doesn't to be used.  If it isn't needed, please can it be removed?

Reply:
Those variables are heavily used to perform integration tests.